app/views/admin/products_v3/_content.html.haml

```diff
@@ -1,5 +1,5 @@
 %turbo-frame#products-content{ target: "_top", refresh: "morph" }
-  .spinner-overlay.hidden
+  .spinner-overlay{ "data-controller": "loading", "data-products-target": "loading", class: "hidden" }
```

Comment:
It would be worth re-basing to revert this on the initial commit. If you're comfortable with interactive rebase, you could try:

```
git rebase -i master
```
You can move this commit up two lines, and change to a "fixup" which will combine it into the previous commit
```
pick 86f88792c2 11058: add sortable products by name
fixup ff97794684 11058: fix specs
pick b104fd8357 11058: add specs
pick 6c65008cf3 11058: add wait for the page to update
```

Reply:
Ah, that's nice. Used it for the first time properly. Would use it often now. Thanks :)

Reply:
A word of caution, only use it when necessary, and only for simple changes if you can. 
It can get very difficult very quickly. I find often I need to `git rebase --abort`!